Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -244,6 +244,20 @@ jobs:
           settingsJson: ${{ env.Settings }}
           secretsJson: ${{ env.RepoSecrets }}
           buildMode: ${{ matrix.buildMode }}
+      
+      - name: Sign
+        if: env.doNotSignApps == 'False' && env.keyVaultCodesignCertificateName != ''
+        id: sign
+        uses: microsoft/AL-Go-Actions/Sign@main
+        with:
+          shell: ${{ needs.Initialization.outputs.githubRunnerShell }}
+          azureKeyVaultURI: ${{ secrets.AZURE_KEYVAULT_URI }}
+          azureKeyVaultClientID: ${{ secrets.AZURE_CLIENT_ID }}
+          azureKeyVaultClientSecret: ${{ secrets.AZURE_CLIENT_SECRET }}
+          azureKeyVaultTenantID: ${{ secrets.AZURE_TENANT_ID }}
+          azureKeyVaultCertificateName: ${{ env.keyVaultCodesignCertificateName }}
```

Comment:
Why not fetch it from `Read settings` step's output?

At some point we'll need to clean up the env vars and keep only what makes sense to be there. `keyVaultCodesignCertificateName` isn't a good candidate for an env variable.

Reply:
You mean the [SettingsJson](https://github.com/microsoft/AL-Go/blob/a83fa796d47309a8a8c586f9e18b42fe66cd2e6b/Actions/ReadSettings/action.yaml#LL41C3-L41C3)? Or add an output to the ReadSettings Action called keyVaultCodesignCertificateName? 

I was trying to avoid the first one, as I think it's kinda unnecessary to pass a json string with all settings just to get one setting

Reply:
Technically, you can do something like `azureKeyVaultCertificateName: ${{ fromJSON(steps.readsettings.SettingsJson).keyVaultCodesignCertificateName }}`, but I agree it isn't pretty.

Separate output for every setting might make the _ReadSettings_ action a bit messy, but isn't it better than having all these environment variables floating around? I fear it'll bite us when we least expect it 😄
Anywho, not a target for your PR. Up to you how you want to pass the cert name.

Reply:
Let's address this in a separate PR :) I suspect we'll have a lot of cleanup to do if we decide to remove settings as env variables